.github/workflows/terraform-checks.yaml

```
@@ -16,7 +16,7 @@ on:
         description: The channel that the charm is deployed from. Defaults to `latest/stable`.
```

please change the description to match :)
```suggestion
        description: The channel that the charm is deployed from. Defaults to `latest/edge`.
```

---

Updated